Description: Update the repository location for MinGW.

  Windows builds failed because this dependency was not at the specified
  location. This patch can be applied to the client submodule to update the
  Dockerfile with a path to MinGW for openSUSE Leap instead. This change has
  been applied upstream in owncloud/client at 6be122e (PR owncloud/client#5900)
  but is not included in the 2.3.3 release.

Last-Update: 2017-10-04
Author: Taylor Smith <tsmith@tsmithcreative.com>

diff --git a/admin/win/docker/Dockerfile b/admin/win/docker/Dockerfile
index 8e40a49..1629b97 100644
--- a/admin/win/docker/Dockerfile
+++ b/admin/win/docker/Dockerfile
@@ -8,7 +8,7 @@ ENV HOME /root
 ENV REFRESHED_AT 20160421

 RUN zypper --non-interactive --gpg-auto-import-keys refresh
-RUN zypper --non-interactive --gpg-auto-import-keys ar http://download.opensuse.org/repositories/windows:/mingw/openSUSE_42.1/windows:mingw.repo
+RUN zypper --non-interactive --gpg-auto-import-keys ar http://download.opensuse.org/repositories/windows:/mingw/openSUSE_Leap_42.1/windows:mingw.repo
 RUN zypper --non-interactive --gpg-auto-import-keys ar http://download.opensuse.org/repositories/isv:ownCloud:toolchains:mingw:win32:2.3/openSUSE_Leap_42.1/isv:ownCloud:toolchains:mingw:win32:2.3.repo
 RUN zypper --non-interactive --gpg-auto-import-keys install cmake make mingw32-cross-binutils mingw32-cross-cpp mingw32-cross-gcc \
                       mingw32-cross-gcc-c++ mingw32-cross-pkg-config mingw32-filesystem \
